src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -90,13 +92,15 @@ const icons = iconsContext.keys().reduce((acc, item) => {
   return acc
 }, {})
 
-function getIconUrl(mimetype) {
+function getIconUrl(file) {
   const keyIcon =
-    getFileTypeFromMime(icons)(mimetype) ||
-    console.warn(
-      `No icon found, you may need to add a mapping for ${mimetype}`
-    ) ||
-    'files'
+    file.type === TYPE_DIRECTORY
+      ? 'folder'
+      : getFileTypeFromMime(icons)(file.mime) ||
+        console.warn(
+          `No icon found, you may need to add a mapping for ${file.mime}`
+        ) ||
+        'files'
 
   return `${window.location.origin}/${icons[keyIcon]}`
```

Comment:
Browsers will add automatically `window.location.origin` if an url starts with a `/`. For instance `/icon` is the same as `localhost/icon`.

Reply:
I think you wrote that line 😛 . But its is correct! The URL we return will be displayed inside an iframe, with a different `window.location.origin`, so it's important to return an absolute URL here.